packages/local_auth/local_auth/pubspec.yaml

```diff
@@ -22,7 +22,7 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  intl: ^0.17.0
+  intl: ">=0.17.0 <0.19.0"
```

Comment:
It looks like `intl` is no longer used directly in this package now that the implementations are fully federated, so this should be removed rather than updated.